services/nuget/nuget-helpers.spec.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const { renderVersionBadge, odataToObject } = require('./nuget-helpers')
+const { test, given } = require('sazerac')
+
+describe('NuGet helpers', function() {
+  test(renderVersionBadge, () => {
+    given({ version: '1.2-beta' }).expect({
+      label: undefined,
```

Comment:
This confused me at first, but it's just because we're not specifying a `feed` argument right?

Reply:
Right. `const { foo, bar } = { foo: 123 }` will give `bar` a value of `undefined`.